main.py

```diff
@@ -376,20 +373,20 @@ def emailLogForCourseID(courseID, recipients):
         return
 
     try:
-        READ_BINARY_MODE = 'rb'
-        logfile = open(getCourseLogFilePath(courseID), mode=READ_BINARY_MODE)
+        logfile = open(getCourseLogFilePath(courseID))
         logContent = logfile.read()
         logfile.close()
     except Exception as exception:
         logger.warning('Exception while trying to read logfile for course {courseID}: {exception}'
                        .format(**locals()))
         return
-    
-    message = MIMEText(logContent,'plain','utf-8')
-    message['From'] = Header(config.Application.Email.SENDER_ADDRESS,'utf-8')
-    message['To'] = Header(', '.join(recipients),'utf-8')
-    message['Subject'] = Header(config.Application.Email.SUBJECT.format(**locals()),'utf-8')
-      
+
+    message = EmailMessage()
+    message['From'] = config.Application.Email.SENDER_ADDRESS
+    message['To'] = ', '.join(recipients)
+    message['Subject'] = config.Application.Email.SUBJECT.format(**locals())
+    message.set_content(logContent)
+
     if options.printEmail is True:
         logger.info("email message: {}".format(message))
```

Comment:
Does this line print the email message correctly if the `Header` constructor isn't used for the various headers?

Reply:
Yes, do you want me to send you sample output from the debugging email setup? I can forward you one of the emails that was sent to me from this.

Reply:
Oh, wait, actually, not sure if I tested this. I will locally.

Reply:
Yes, this still works and shows the headers when you use the flags `--email` and `--printEmail`.